packages/ui/src/components/Command/CommandMenu.tsx

```diff
@@ -246,9 +244,7 @@ const CommandMenu = ({ projectRef }: CommandMenuProps) => {
               {site === 'studio' && (
                 <CommandGroup heading="Navigate">
                   {sharedItems.tools.map((item) => {
-                    const itemUrl = (
-                      projectRef ? item.url.replace('_', projectRef) : item.url
-                    ).split('https://supabase.com/dashboard')[1]
+                    const itemUrl = projectRef ? item.url.replace('_', projectRef) : item.url
```

Comment:
I think this .split(supabase.com/dashboard) bit was a leftover from a long time ago. 

We used to spell out the entire url: 
https://app.supabase.com/_/editor

but then we switched to: 
/project/_/editor

looks like we didn't update this in the process